grades/api.py

```diff
@@ -165,6 +167,8 @@ def freeze_user_final_grade(user, course_run, raise_on_exception=False):
     try:
         CachedEdxDataApi.update_all_cached_grade_data(user)
     except Exception as ex:  # pylint: disable=broad-except
+        con = get_redis_connection("redis")
+        con.lpush('{}'.format(course_run.edx_course_key), user.id)
```

Comment:
Oh, I think I was thinking maybe to prefix it with something. But I don't know if there is any benefit in doing that. So I'll remove it